schema/scope_test.go

```diff
@@ -498,3 +498,42 @@ func TestApplyingExternalNamespaceToNonRefTypes(t *testing.T) {
 		})
 	}
 }
+
+func TestMismatchedRoot(t *testing.T) {
+	// This is a common user mistake: invalid root key
```

Comment:
Some clarity on what is mismatched here would help me if I ever revisited this test, something like (maybe this is too implementation specific):

> Error when a Scope's RootValue is not a key in its ObjectsValue.

Reply:
Okay. I clarified it.